### PR TITLE
Nav bar fixes 

### PR DESCRIPTION
**Pull request by flyingboat505**
Removed Extraneous stuff (Double check sign out)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,14 +7,14 @@
     "@testing-library/react": "^9.5.0",
     "@testing-library/user-event": "^7.2.1",
     "axios": "^0.19.2",
-    "bootstrap": "^4.5.0",
+    "bootstrap": "^3.4.1",
```
Comment (alizand1992) [inline package.json]: Please do not downgrade the bootstrap. We are using bootstrap 4 and not 3. Revert this change.

Additionally please remove package-lock.json. this project uses Yarn and this is resulting in build failures

File: package.json [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
     "react": "^16.13.1",
     "react-bootstrap": "^1.3.0",
     "react-dom": "^16.13.1",
     "react-loading": "^2.0.3",
     "react-redux": "^7.2.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1",
+    "react-scripts": "^3.4.3",
```
Comment (alizand1992) [inline package.json]: Why was this changed?

File: src/Components/Common/Menu.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      </Form>
+    </Navbar.Collapse>
+    </Navbar> 
+  );
+};
+
+export default Menu;
+
+/*
+<Link to="/user/sign_in">Sign In</Link> <br />
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: Remove all unnecessary comments from code

File: src/Components/Common/Menu.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+          <Link to="/user/sign_up">Sign Up</Link> <br />
+          <Link to="/user/sign_out">Sign Out</Link> <br />
+          <Link to="/user/profile">Profile</Link> <br />
+          <Link to="/notifications">Notifications</Link> <br />
+          <Link to="/notifications/new">Create Notification</Link> <br />
+*/
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: Missing new line in the end of the file

File: src/Components/Common/Menu.js [hunk 5]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Navbar, Nav, Button, NavDropdown, Form, FormControl } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+const Menu = () => {
```
Comment (alizand1992) [inline src/Components/Common/Menu.js:5]: needs a space between imports and the function declaration

File: src/Components/Common/Menu.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <Navbar bg="light" expand="lg">
+    <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: indentation is messed uo

File: src/Components/Common/Menu.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+        <NavDropdown title="User" id="basic-nav-dropdown">
+          <NavDropdown.Item as={Link} to="/user/Profile">Profile</NavDropdown.Item>
+          <NavDropdown.Divider />
+          <NavDropdown.Item href="#action/3.4">About Us</NavDropdown.Item>
+        </NavDropdown>
+
+        <NavDropdown title="Notifications" id="basic-nav-dropdown">
+          <NavDropdown.Item as={Link} to="/notifications">List Notifications</NavDropdown.Item>
+          <NavDropdown.Item as={Link} to="/notifications/new">Create Notification</NavDropdown.Item>
+
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: Vertical space

File: src/Components/Common/Menu.js [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+        </NavDropdown>
+
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: vertical space

File: src/Components/Common/Menu.js [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+      </Nav>
+      <Form inline>
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: We don't have a search functionality. Why is this here?

File: src/Components/Common/Menu.js [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+    <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
+    <Navbar.Toggle aria-controls="basic-navbar-nav" />
+    <Navbar.Collapse id="basic-navbar-nav">
+      <Nav className="mr-auto">
+        <Nav.Link as={Link} to="/user/sign_in">Sign In</Nav.Link>
+        <Nav.Link as={Link} to="/user/sign_out">Sign Out</Nav.Link>
+        <NavDropdown title="User" id="basic-nav-dropdown">
+          <NavDropdown.Item as={Link} to="/user/Profile">Profile</NavDropdown.Item>
+          <NavDropdown.Divider />
+          <NavDropdown.Item href="#action/3.4">About Us</NavDropdown.Item>
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: Why is about us under profile? Why are sign in and sign out arent

File: src/Components/Common/Menu.js [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+import { Navbar, Nav, Button, NavDropdown, Form, FormControl } from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+const Menu = () => {
+  return (
+    <Navbar bg="light" expand="lg">
+    <Navbar.Brand href="#home">React-Bootstrap</Navbar.Brand>
+    <Navbar.Toggle aria-controls="basic-navbar-nav" />
+    <Navbar.Collapse id="basic-navbar-nav">
+      <Nav className="mr-auto">
+        <Nav.Link as={Link} to="/user/sign_in">Sign In</Nav.Link>
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: Why is sign in always showing? it should hide when the user is signed in already

File: src/Components/Common/Menu.js [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+        <Nav.Link as={Link} to="/user/sign_out">Sign Out</Nav.Link>
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: Sign out should not show unless the user is signed in

File: src/Components/Common/Menu.js [hunk 13]
```diff
@@ -0,0 +1,44 @@
...
+        <NavDropdown title="User" id="basic-nav-dropdown">
+          <NavDropdown.Item as={Link} to="/user/Profile">Profile</NavDropdown.Item>
```
Comment (alizand1992) [inline src/Components/Common/Menu.js]: Missing Sign Up link

File: src/Components/Common/Menu.js (hunk 6 above)
Comment (alizand1992) [inline src/Components/Common/Menu.js]: Why does this say React-Bootstrap. Please use our project name for this. And fix the href to point to the a real page instead #home.

**Comment by flyingboat505** [comment]
Closed on request
